stlTranslator/SourceFiles/triangulation.cpp

```diff
@@ -33,5 +33,6 @@ vector<Triangle> Triangulation::createTriangles(const string& data)
             indices.clear();
```

Comment:
No need to clear the vector, as it is reset with each loop iteration.

Reply:
I am clearing it inside the loop as I want at most nine members at a time.
Also the declaration of vector indices is outside the loop so it will not reset for each iteration.